ToyBox/classes/Infrastructure/CasterHelpers.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Kingmaker.Blueprints;
+using Kingmaker.Blueprints.Classes;
+using Kingmaker.Blueprints.Classes.Selection;
+using Kingmaker.Blueprints.Classes.Spells;
+using Kingmaker.UnitLogic;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ToyBox.classes.Infrastructure {
+    public static class CasterHelpers {
```

Comment:
please consider calling this CasterUtils to follow the convention in other files